config/vendor/go4.org/errorutil/highlight.go

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 */
 
 // Package errorutil helps make better error messages.
-package errorutil
+package errorutil // import "go4.org/errorutil"
```

Comment:
Its not my comment, its in upstream's source. https://godoc.org/go4.org/errorutil